src/ILogger/ApplicationInsightsLogger.cs

```diff
@@ -86,16 +86,22 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                         formatter(state, exception),
                         ApplicationInsightsLogger.GetSeverityLevel(logLevel));
                     this.PopulateTelemetry(traceTelemetry, state, eventId);
+                    if (exception != null)
+                    {
+                        traceTelemetry.Properties.Add("ExceptionMessage", exception.Message);
+                    }
+
                     this.telemetryClient.TrackTrace(traceTelemetry);
                 }
                 else
                 {
                     ExceptionTelemetry exceptionTelemetry = new ExceptionTelemetry(exception)
```

Comment:
this branch seems to be executed when exception == null  && this.applicationInsightsLoggerOptions.TrackExceptionsAsExceptionTelemetry (else), but it does not seem right to track exception and  Message = exception.Message will throw. Am I missing something?

Reply:
discussed offline. this is not the case.